src/appliance-onboarding-script/appliance_setup/avs/avsarconboarder/utils/user_config_validatator.py

```diff
@@ -74,6 +101,60 @@ def block_DHCP_config(self):
         if "isStatic" in self.__config and not self.__config["isStatic"]:
             raise InvalidInputError("isStatic should be true. Only Static IP configuration is supported currently")
 
+    def check_if_old_nw_config(self):
+        if not self.__config["applianceControlPlaneIpAddress"].strip():
+            return False
+        if not self.__config["staticIpNetworkDetails"]["networkForApplianceVM"].strip():
+            return False
+        if not self.__config["staticIpNetworkDetails"]["networkCIDRForApplianceVM"].strip():
+            return False
+        if not self.__config["staticIpNetworkDetails"]["k8sNodeIPPoolStart"].strip():
+            return False
+        if not self.__config["staticIpNetworkDetails"]["k8sNodeIPPoolEnd"].strip():
+            return False
+        if not self.__config["staticIpNetworkDetails"]["gatewayIPAddress"].strip():
+            return False
+        return True
+
+    def check_if_new_nw_config(self):
+        if self.__config["applianceControlPlaneIpAddress"].strip():
+            return False
+        if not self.__config["staticIpNetworkDetails"]["networkForApplianceVM"].strip():
+            return False
+        if not self.__config["staticIpNetworkDetails"]["networkCIDRForApplianceVM"].strip():
+            return False
+        if self.__config["staticIpNetworkDetails"]["k8sNodeIPPoolStart"].strip():
+            return False
+        if self.__config["staticIpNetworkDetails"]["k8sNodeIPPoolEnd"].strip():
+            return False
+        if self.__config["staticIpNetworkDetails"]["gatewayIPAddress"].strip():
+            return False
+        return True
+
+    def get_config_version(self):
+        if(self.check_if_old_nw_config()):
+            return Constant.OLD_CONFIG_VERSION
+        if(self.check_if_new_nw_config()):
+            return Constant.NEW_CONFIG_VERSION
```

Comment:
>NEW_CONFIG_VERSION [](http://example.com/codeflow?start=28&length=18)

[Naming]

For this and other places - function names and var names should be of the format function-config-v1 instead of function-new-config. In case we have yet another new config version, how would we name the functions/ vars then?

Reply:
Yes..This is great idea. I will incorporate the change.